### DISCUSSION TITLE
Bay Area meetup at Parker?

### DISCUSSION THREAD

**Discussion by wbpaley** [bevyengine/bevy#11413] [Ideas]
I wonder if there's interest in a meetup event at my building near the waterfront in Oakland California?

http://ParkerElectricFoundation.org

I'd be pleased to host people (gratis, of course). Honored if Cart's ever in the neighborhood...

(Could be a demo party/exchange: I have a bright laser projector w/15' screen, killer stereo, and three beers and a cider on tap.)

**Comment by atlv24** [comment]
I'm nearby and interested.

**Reply by wbpaley** [reply]
Neat! Let's see if a quorum responds here. 

Or are there other ways to invite people & settle on a date? (I'm not that kind of "influencer"...  ; )
